lib/ex_doc/formatter/html.ex

```diff
@@ -10,10 +10,18 @@ defmodule ExDoc.Formatter.HTML do
   @main "overview"
 
   @doc """
-  Generate HTML documentation for the given modules
+  Generate HTML documentation for the given modules.
+
+  Please do not call this function directly. Use `ExDoc.generate_docs/3` instead.
   """
```

Comment:
should we set this to `@docfalse` ?
I think it's good to have it, just to let the developers there is a formatter here,
but maybe we could just move that information to `@moduledoc`